src/modules/apiCall.js

```diff
@@ -0,0 +1,17 @@
+const dataJson = [''];
+const apiCall = async (homePageOption) => {
+  await fetch(`https://api.tvmaze.com/search/shows?q=${homePageOption[0]}`)
+    .then((data) => data.json())
+    .then((data) => {
+      dataJson.length = 0;
```

Comment:
- [ ] Please make sure to fix this error and make sure that you really know how the API response data looks like, here it looks like it is a string 

![image](https://user-images.githubusercontent.com/74744787/211094140-70fdf244-ddd6-4476-97de-68f2c71a5a33.png)

Reply:
The error is now solved.